### PR TITLE
fix(VSparkline): trend should respect label-size prop

### LINKED ISSUE #6672: [Bug Report] Sparkline doesn't support label-size for 'trend' type
### Versions and Environment
**Vuetify:** 1.5.4
**Vue:** 2.6.8
**Browsers:** Chrome 72.0.3626.119
**OS:** Mac OS 10.14.3

### Steps to reproduce
Set `label-size` attr for `trend` type to any value

### Expected Behavior
Font size for labels must be changed

### Actual Behavior
Font size for labels doesn't change

### Reproduction Link
<a href="https://codepen.io/kerf/pen/VRmLzO" target="_blank">https://codepen.io/kerf/pen/VRmLzO</a>

### PR DESCRIPTION
**Pull request by nekosaur**
## Description
sparkline did not use label-size prop when in trend mode

## Motivation and Context
closes #6672

## How Has This Been Tested?
playground, unit test

## Markup:

<details>

```vue
<template>
<v-sparkline
    :value="value"
    label-size="14"
  >
    <template v-slot:label="item">
      ${{ item.value }}
    </template>
  </v-sparkline>
</template>

<script>
export default {
  data: () => ({
    value: [0, 5, 10, 2, 20]
  })
}
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by johnleider** [comment]
I may have goofed this by merging the other sparkline PR first.

File: packages/vuetify/src/components/VSparkline/VSparkline.ts [hunk 1]
```diff
@@ -409,7 +409,7 @@ export default mixins<options &
         },
       }), [
         this.genGradient(),
-        this.hasLabels && this.genLabels(-2),
+        this.hasLabels && this.genLabels(-(this._lineWidth / 2)),
```
Comment (sh7dm) [inline packages/vuetify/src/components/VSparkline/VSparkline.ts]:
```suggestion
        this.hasLabels && this.genLabels(-this._lineWidth / 2),
```

File: packages/vuetify/src/components/VSparkline/__tests__/VSparkline.spec.ts [hunk 2]
```diff
@@ -213,4 +213,16 @@ describe('VSparkline.ts', () => {
 
     expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('should render component with label size and match a snapshot', async () => {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VSparkline/__tests__/VSparkline.spec.ts]:
```suggestion
  it('should render component with label size and match a snapshot', () => {
```

**Comment by johnleider** [comment]
Looking at this and comparing before and after for the provided template doesn't seem to have any changes?

**Comment by nekosaur** [comment]
Hmm yeah I must have fixed this in one of the other sparkline PRs. closing.

**Pull request closed without merging**